.github/workflows/go-tests.yml

```diff
@@ -37,6 +37,15 @@ jobs:
           go install gotest.tools/gotestsum@latest
           make test-certs
 
+      # install nomad
+      - name: Install nomad
+        run : |
+          sudo apt -y install wget gpg coreutils
+          wget -O- https://apt.releases.hashicorp.com/gpg | gpg --dearmor -o /usr/share/keyrings/hashicorp-archive-keyring.gpg
+          echo "deb [signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/hashicorp.list
+          sudo apt update && sudo apt -y install nomad
```

Comment:
Do we want to pin this to a version, or does that not really add anything nor matter?

Reply:
I thought about this, but i actually want latest stable. Because if our tests start failing on latest stable, I wanna know.